### PR TITLE
Regenerate types

### PR DESCRIPTION
**Pull request by michaelbausor**
- Pubsub updates
- New vision types

### CODE REVIEW

File: src/pubsub/v1/pubsub.php [hunk 1]
```diff
@@ -1,37 +1,7 @@
...
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
```
Comment (garrettjonesgoogle) [inline src/pubsub/v1/pubsub.php]: Losing the license doesn't seem good...

Comment (michaelbausor) [inline src/pubsub/v1/pubsub.php]: All other files don't have licenses - this is a holdover because pubsub was the first published.

Comment (garrettjonesgoogle) [inline src/pubsub/v1/pubsub.php]: Alrighty.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**